ostia-operator/pkg/apis/ostia/v1alpha1/types.go

```diff
@@ -25,9 +25,10 @@ type API struct {
 
 // APISpec Contains the Spec of the API object
 type APISpec struct {
-	Expose    bool       `json:"expose"` //TODO: Make expose readonly after creation
-	Hostname  string     `json:"hostname"`
-	Endpoints []Endpoint `json:"endpoints"`
+	Expose     bool        `json:"expose"` //TODO: Make expose readonly after creation
```

Comment:
Just for the future: I think git changes should be minimal as possible, so re indenting struct because of adding new field goes totally against that. We should adopt a code style that avoids that.

Reply:
IMO one of the nice things about golang is the standardisation on formatting, so running `go fmt -w` on some file will inevitably lead to changes like this and I'm not sure that is avoidable?

Reply:
I agree that formatting everything with gofmt is nice, but this code style makes figuring out what is just whitespace change and what is actual change in the diff hard. It makes reviews harder. It makes a lot collateral damage.
Other languages try to avoid this by using trailing comas in hashes so they don't affect previous line when adding new element. This looks like madness to me. I get it is the default behaviour, but I seriously hope we can adopt different style and keep the collateral changes to minimum.

Reply:
Mark the "Hide whitespace changes" in diff settings.. that way it doesn't hurt that much. :) 

(There's a way to activate it by default or in a project basis?)